### DISCUSSION TITLE
How to use Airflow Stable Rest API [Airflow version 2.0.0] deployed on GCP Cloud Composer

### DISCUSSION THREAD

**Discussion by achoudha-tibco** [apache/airflow#17499] [Q&A]
Hi team,

Env : We using GCP cloud composer to run airflow dags [Airflow version 2.0.0 +]. 
Aim : To use apache airflow stable apis to trigger dags externally using REST.

In Airflow vrsion <2.0.0
We tried to use airflow experimental apis to trigger dags externally by setting airflow override configuration in 
cloud composer: airflow.api.auth.backend.default and making IAP request. It worked fine.

In Airflow version >2.0.0
The new stable api doesn't supports airflow.api.auth.backend.default. Hence it is throwing :403 FORBIDDEN upon making an valid IAP request.

**Q1.** Do we need to do double Auth i.e first Authorization with google IAP and then with Airflow ?

I found similar use case with Airflow RBAC UI where only google auth is enough, by providing some airflow override configuration in Cloud composer. [https://www.youtube.com/watch?v=E8ouMCCtq9g ]
**Q2.** Do we have similar configuration to access stable apis ?

**Comment by potiuk** [comment]
I think you should ask in the Composer support group
